audnauseum/audio_tools/recorder.py

```diff
@@ -62,14 +62,18 @@ def audio_callback(self, indata, frames, time, status):
 
     def on_rec(self):
         current_block = self.loop.audio_cursor
+
         print(f'Audio Cursor at Record press: {current_block}')
         self.recording = True
         # create directory if not present
         if not os.path.exists(self.directory):
             os.makedirs(self.directory)
+        now = None
+        filename = None
```

Comment:
These variables are getting assigned just below, they shouldn't need to be assigned `None` beforehand.

```suggestion
```